roles/netbootxyz/defaults/main.yml

```diff
@@ -369,21 +369,18 @@ releases:
     base_dir: pub/OpenBSD
     enabled: true
     menu: bsd
-    mirror: http://ftp.openbsd.org
+    mirror: https://cdn.openbsd.org
     name: OpenBSD
     versions:
-    - code_name: '7.4'
-      image_ver: '74'
+    - code_name: '7.6'
+      image_ver: '76'
+      name: '7.6'
+    - code_name: '7.5'
+      image_ver: '75'
       name: '7.4'
```

Comment:
Switch this to 7.5 as well, but other than that LGTM.

Reply:
done!